.pre-commit-config.yaml

```diff
@@ -1,19 +1,19 @@
-# See https://pre-commit.com for more information
```

Comment:
Oh, shoot. You already did this! Gaaah. Are you fine with just `black` instead of `darker`? Do you have a preference. Sorry to miss this!

Reply:
From what I can tell when black-jupyter pre-commit runs, it will clean up an entire file. That could lead to a lot of git blames on unrelated changes.

I'm all for blacking these up, but should we consider including a full black commit on each repo that is excluded from git blame?

Something like this https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html#avoiding-ruining-git-blame

Reply:
I like this approach - thanks for talking through it with me on Tuesday. Here's my plan:

* I'll preemptively squash the current commits into a single one (because Squash and Merge isn't an option here). I know it's a bit bleugh to force push once stuff is in PR, but I think this time makes sense, where the later commits can't be squashed.
* I'll use black over all Python files (including test files). This will be it's own commit, for clarity.
* Then I'll add the secondary commit to ignore the `black` commit when using `git blame` (this is really cool by the way).
* Then I'll push up and we'll be set. 🤞

Other things:

* I feel like if there is any change in the service code, we probably should have an associated patch release, because even though the code _should_ be unchanged functionally, I feel like we need to test this in isolation. Does this sound reasonable, or am I just being paranoid?
* Later work should, in the near future, add these checks to CI/CD, as we discussed at some point in the last couple of days.
* I'll try to prioritise HyBIG, because I think that's the repository where you'll be working next.

Does that all sound good?